src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Appointment.kt

```diff
@@ -46,6 +46,6 @@ enum class Attended {
 
   @JsonValue
   open fun toLower(): String? {
-    return this.toString().lowercase()
+    return this.toString().toLowerCase()
```

Comment:
this method is deprecated since kotlin 1.5 - the existing method is the newer version